xilem_core/examples/filesystem.rs

```diff
@@ -166,11 +166,11 @@ impl<State, Action> View<State, Action, ViewCtx> for File {
     ) -> Mut<'el, Self::Element> {
         if prev.name != self.name {
             let new_path = ctx.current_folder_path.join(&*self.name);
-            let _ = std::fs::rename(&element, &new_path);
+            let _ = std::fs::rename(&*element, &new_path);
```

Comment:
:+1:  (I was already about to do this as PR, as I was annoyed in my editor, but was too lazy...)